web/WebDevelopersGuide.md

```diff
@@ -599,7 +605,32 @@ return a new Sort instance.
 ###### Properties
  * `Column column` - The column which is sorted.
  * `String direction` - The direction of this sort, either `ASC`, `DESC`, or `REVERSE`.
- * `boolean isAbs` - True if the absolute value of the column should be used when sorting; defaults to false.
+ * `boolean isAbs` - True if the absolute value of the column should be used when sorting; defaults to false. 
+
+##### Abstract Class `CustomColumn`
+ * `String method` - The type of select method to use, either `VIEW` or `SELECT`. Defaults to `VIEW`.
+
+##### Class `CustomColumnFormula` extends `CustomColumn`
+* `String name` - The name of the custom column to create.
+* `String formula` - The formula to use for this custom column.
+
+##### Class `CustomColumnColor` extends `CustomColumn`
+ * `String name` - The name of the column to color. Set to `null` to apply to all columns (color entire row).
+ * `String condition` - The condition in which to apply this color.
+ * `String color` - The hex representation of the color to apply, or case-insensitive CSS color module name (see https://drafts.csswg.org/css-color/#named-colors)
+ * `String type` - The type of cell color formatting, `BACKGROUND` or `FOREGROUND`.
+
+##### Class `CustomColumnDateFormat` extends `CustomColumn`
+ * `String name` - The name of the column to apply date formatting.
+ * `String dateFormat` - Any of the [Java DateFormat](https://docs.oracle.com/javase/7/docs/api/java/text/DateFormat.html) strings.
```

Comment:
pointer to dh.i18n too? since it is going to be reflected back to the JS API for use locally, won't be evaluated on the server